apigw-lambda-translate-cdk/example-pattern.json

```diff
@@ -0,0 +1,56 @@
+{
+    "title": "API Gateway - Lambda - Translate",
+    "description": "Perform language translation using Translate through Lambda and API Gateway",
+    "language": "Python",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "This patterns shows CDK deployment on how to leverage Amazon API Gateway, AWS Lambda, and Amazon Translate to perform language translation in a serverless fashion."
+      ]
+    },
+    "gitHub": {
+      "template": {
+        "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/apigw-lambda-translate-cdk",
+        "templateURL": "serverless-patterns/apigw-lambda-translate-cdk",
+        "projectFolder": "apigw-lambda-translate-cdk",
+        "templateFile": "app.py"
+      }
+    },
+    "resources": {
```

Comment:
please add a link to API GW documentation as well

Reply:
Done. Added documentation links for developer guide and features of API Gateway.